### ISSUE TITLE
1846 2p - ERIE's second token should be placed in Erie during setup if ERIE is removed from the game

### ISSUE THREAD

**Issue by michaeljb** [tobymao/18xx#1781]
When ERIE is removed, its second token will be placed in Erie, so its reservation needs to stick around so that two other corporations may not token in Erie. When two corporations have incorrectly tokened there, when a green tile is placed, an error is thrown as there is no slot for ERIE to put its second token down.

https://www.18xx.games/game/11557?action=187

**Comment by Auzzy** [comment]
I do not believe this is correct. According to rule 6.53 (of the GMT version): "Before phase IV, **if the corporation with a reserved city spot is in play** (and hasn’t closed, see section 8), other corporations may not place a token in that spot." (emphasis mine). My understanding is that a company which has been removed is not in play, and thus the company does not retain its reservation.

So in the scenario you describe, I think the problem is that the system is trying to retain Erie's reservation when it should not.

**Comment by michaeljb** [comment]
This problem is specific to the 2p variant.

In the 2p variant, the removed corporation places their home token, and later places a second token in a particular hex when a green tile is laid on that hex.

ERIE is the only corporation where that specified second hex starts with two available city slots, instead of one.

In the linked game, NYC's home token was placed in Erie, and then PRR placed a token in the second Erie slot. 

The NilClass error that Michal mentions in the chat in the linked game was caused by attempting to upgrade Erie to a green tile, at which point ERIE tries to place a token there, but finds no slot--the system is definitely not trying to retain Erie's reservation.

While the 2p Variant rules don't explicitly say that ERIE's reservation is preserved as a special exception, if that is not the case, then the token slots in Erie can fill up, and ERIE will be unable to place its second token (as is the case in game 11557). Therefore I have inferred that ERIE's reservation should be preserved. In order to be certain, I have posted this question on BGG - https://boardgamegeek.com/thread/1616729/article/35960377#35960377
